assisted_deployment.sh

```diff
@@ -7,18 +7,181 @@ source common.sh
 source assisted_deployment_utils.sh
 
 
+ASSISTED_INSTALL_TEST_INFRA=${ASSISTED_INSTALL_TEST_INFRA:-}
+ASSISTED_INSTALLER_NAMESPACE="${ASSISTED_INSTALLER_NAMESPACE:-assisted-installer}"
+ASSISTED_OPERATOR_INDEX="${ASSISTED_OPERATOR_INDEX:-quay.io/ocpmetal/assisted-service-index:latest}"
```

Comment:
Should we have also an override for OpenShift Versions to put in the subscriptions?

Reply:
Do you mean as the target openshift version for AI? Or the `OPENSHIFT_VERSIONS` var?

Prob worth it, yeah.

Reply:
The OPENSHIFT_VERSIONS var, since I see people tweaking it all the time